CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 All changes that impact users of this module are documented in this file, in the [Common Changelog](https://common-changelog.org) format with some additional specifications defined in the CONTRIBUTING file. This codebase adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## Unreleased [patch]
+
+_Full changeset and discussions: [#1040](https://github.com/OpenTermsArchive/engine/pull/1040)._
+
+> Development of this release was made on a volunteer basis by [@jetlime](http://github.com/jetlime).
+
+### Added
+
+- Link the contributing file to the readme of the repository.
```

Comment:
As you noted, this is an internal change, hence no modification to the changelog is required. Currently, there isn't a mechanism in place to bypass the changelog verification. Therefore, even if it fails, we'll proceed with merging the change.

```suggestion
## Unreleased
```